package.json

```diff
@@ -1,75 +1,16 @@
 {
-  "name": "react-simple-boilerplate",
-  "version": "1.0.0",
-  "description": "Simple react boilerplate with support for travis ci, webpack, sass etc.",
-  "main": "index.js",
-  "scripts": {
-    "build": "npm run build:css && webpack ---config ./config/webpack.prod.config.js",
-    "dev": "npm run watch:css && webpack-dev-server --config ./config/webpack.dev.config.js --open",
-    "lint": "./node_modules/.bin/eslint --ext .js,.jsx src/",
-    "lint:fix": "./node_modules/.bin/eslint --ext .js,.jsx src/ --fix",
-    "build:css": "postcss src/assets/css/styles.css -o src/assets/css/main.css",
-    "watch:css": "postcss src/assets/css/styles.css -o src/assets/css/main.css",
-    "test": "cross-env jest --collect-coverage --env=jsdom",
-    "test:watch": "jest --watch"
-  },
-  "husky": {
-    "hooks": {
-      "pre-commit": "lint-staged"
-    }
-  },
-  "lint-staged": {
-    "*.{js,jsx}": "./node_modules/.bin/eslint --ext .js,.jsx src/ && cross-env jest --collect-coverage --env=jsdom"
-  },
-  "keywords": [
-    "react",
-    "boilerplate",
-    "webpack",
-    "webpack hmr",
-    "travis",
-    "react-simple-boilerplate",
-    "react-boilerplate"
+  "private": true,
+  "workspaces": [
```

Comment:
Let’s add specific packages. Instead of ‘packages/*’

Reply:
In this case it’s boilerplate and website

Reply:
I tried was experiencing issues while running root level scripts based on lerna. 
Moreover, that is probably not related to the scope of this PR.

Reply:
> In this case it’s boilerplate and website

For this you might also need to rename the boilerplate package in its package.json